docs/configure/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Overview'
+---
+
+Storybook is configured via a folder, called `.storybook` which contains various configuration files. 
+
+> Note you can change the folder that Storybook uses by setting the `-c` flag to your `start-storybook` and `build-storybook` scripts.
+
+
+### Configure your Storybook project
+
+The main configuration file is `main.js`. This file controls the behaviour of the Storybook server, and so you must restart Storybook’s process when you change it. It contains the following:
+
+```js
+// .storybook/main.js
+
+module.exports = {
+  stories: ['../src/**/*.stories.(js|mdx)'],
+  addons: ['@storybook/addon-essentials']
+}
+```
+
+The `main.js` configuration file is a [preset](../presets/introduction) and as such has a powerful interface, but the key fields within it are:
+
+- `stories` - a glob that indicates the [location of your story files](#configure-story-loading), relative to `main.js`.
```

Comment:
it's an array of globs

Reply:
Fixed with the latest commit, thank you for the feedback @ndelangen